bower.json

```diff
@@ -41,7 +41,7 @@
         "xpath": "dimagi/js-xpath#v0.0.3",
         "jsdiff": ">=1",
         "markdown-it": "~7.0.0",
-        "codemirror": "3.24.0"
+        "codemirror": "5.19.0"
```

Comment:
5.20.2 is out, but the owners have taken a stance against bower https://github.com/codemirror/CodeMirror/issues/4328